common/src/main/java/eu/europeana/cloud/common/model/dps/NodeStatistics.java

```diff
@@ -1,27 +1,58 @@
 package eu.europeana.cloud.common.model.dps;
 
+import javax.xml.bind.annotation.XmlRootElement;
 import java.util.HashSet;
 import java.util.Set;
 
 /**
  * Statistics for a node.
  */
+@XmlRootElement()
 public class NodeStatistics {
```

Comment:
General remark: I think the order of elements within the class should be changed. Fields should be before the methods that use them. Now it looks messy.